deploy/chaos/loadtest/locustfile.py

```diff
@@ -227,15 +228,54 @@ def create_channel(self):
             headers={
                 "content-type": "application/json",
                 'X-CSRFToken': self.client.cookies.get('csrftoken'),
+                'Referer': self.client.base_url
             }
         )
 
         data = resp.json()
-        self.created_channels.append(data['id'])
+        channel_id = data["id"]
+
+        try:
+            copy_data = {
+                # KA Computing root node.
+                "node_ids": ["76d5fd8636004b459a09aecbb2f8294e"],
+                "sort_order": 1,
+                "target_parent": data["main_tree"]["id"],
+                "channel_id": channel_id
+            }
+
+            copy_resp = self.client.post("/api/duplicate_nodes/",
+                                         data=json.dumps(copy_data),
+                                         headers={
+                                             "content-type": "application/json",
+                                             'X-CSRFToken': self.client.cookies.get('csrftoken'),
+                                             'Referer': self.client.base_url
+                                         })
+            copy_resp_data = copy_resp.json()
+            task_id = copy_resp_data["id"]
+            finished = False
+            time_elapsed = 0
+            while not finished:
```

Comment:
Not needed rn, but a future TODO would be to move this task querying code as another function so it can easily be reused. Potentially as another locust task even!

Reply:
Thanks for the suggestion! I added a `_run_async_task` method to `BaseTaskSet` so that any locust task can run an async task just by passing in the URL, channel ID, and relevant data.